<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Colorer</class>
 <widget class="QWidget" name="Colorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Color balance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QSlider" name="redSlider">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Red:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Green:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Blue:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QSlider" name="blueSlider">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="greenSlider">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="redLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="greenLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="blueLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="balanceButton">
          <property name="text">
           <string>Autobalance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Threshold:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="thresholdSlider">
          <property name="maximumSize">
           <size>
            <width>79</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thresholdLabel">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="greyButton">
        <property name="text">
         <string>&quot;Gray world&quot; method</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Hue/Saturation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QSlider" name="hueSlider">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Hue:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Saturation:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Brightness:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QSlider" name="lightnessSlider">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="satuSlider">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="hueLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="satuLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="lightnessLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="invertButton">
          <property name="text">
           <string>Invert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnwButton">
          <property name="text">
           <string>Convert to grayscale</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
